packages/fiori/src/DynamicPageTitle.ts

```diff
@@ -134,7 +134,7 @@ class DynamicPageTitle extends UI5Element {
 	 * @public
 	 */
 	@slot({ type: HTMLElement })
-	navigationActions!: HTMLElement[];
+	navigationActions!: Array<Toolbar>;
```

Comment:
actions and navigationActions as names imply that consumers have to pass actions, buttons. However, looking at the provided samples, the DynamicPageTitle rather expects and works with a Toolbar component. If that's the case, actionBar and navigationBar could be more appropriate. 
actions and navigationActions are perfect if the consumer pass buttons.

Reply:
Correct, we talked to change the names to actionBar, navigationBar and to remove the redundant exapndedHeading and exapandedContent slots. Another PR is in progress.